server/game/cards/01_SOR/leaders/SabineWrenGalvanizedRevolutionary.ts

```diff
@@ -0,0 +1,36 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { LeaderUnitCard } from '../../../core/card/LeaderUnitCard';
+
+export default class SabineWrenGalvanizedRevolutionary extends LeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '4841169874',
+            internalName: 'sabine-wren#galvanized-revolutionary',
+        };
+    }
+
+    protected override setupLeaderSideAbilities() {
+        this.addActionAbility({
+            title: 'Deal 1 damage to each base',
+            cost: AbilityHelper.costs.exhaustSelf(),
+            immediateEffect: AbilityHelper.immediateEffects.damage((context) => ({
+                amount: 1,
+                target: context.game.getPlayers().map((player) => player.base)
+            }))
+        });
+    }
+
+    protected override setupLeaderUnitSideAbilities() {
+        this.addOnAttackAbility({
+            title: 'Deal 1 damage to each enemy base',
+            immediateEffect: AbilityHelper.immediateEffects.damage((context) => ({
+                amount: 1,
+                target: context.game.getPlayers()
+                    .filter((player) => player !== context.player)
```

Comment:
For simplicity, maybe just `target: context.player.opponent.base`? We've been making the assumption that it's only two players pretty much everywhere so might as well leverage it here

Reply:
I figured it wouldn't hurt to make it multiplayer compatible ahead of time, but if you say so I'll change it.